sdk/servicebus/azure-servicebus/README.md

```diff
@@ -227,20 +357,47 @@ special "dead-letter sub-queue" where it can be accessed using the `get_queue_de
 
 There are various timeouts a user should be aware of within the library.
 - 10 minute service side link closure:  A link, once opened, will be closed after 10 minutes idle to protect the service against resource leakage.  This should largely
-be transparent to a user, but if you notice a reconnect occuring after such a duration, this is why.  Performing any operations, including management operations, on the
+be transparent to a user, but if you notice a reconnect occurring after such a duration, this is why.  Performing any operations, including management operations, on the
 link will extend this timeout.
 - idle_timeout: Provided on creation of a receiver, the time after which the underlying UAMQP link will be closed after no traffic.  This primarily dictates the length
 a generator-style receive will run for before exiting if there are no messages.  Passing None (default) will wait forever, up until the 10 minute threshold if no other action is taken.
-- max_wait_time: Provided when calling receive() to fetch a batch of messages.  Dictates how long the receive() will wait for more messages before returning, similarly up to the aformentioned limits.
+- max_wait_time: Provided when calling receive() to fetch a list of messages.  Dictates how long the receive() will wait for more messages before returning, similarly up to the aformentioned limits.
```

Comment:
My fear with as your first sentence is they will think "if messages come forever then does it keep receiving forever."  However, the second sentence is a good clarification. Howabout combining my first clause (how long receive will wait for more messages before returning; perhaps with the addition of "at most" at the start) with your 2nd clause to then specify the special behavior that it will return as soon as at least one message is received within that timeout.